src/platform/forms-system/src/js/definitions/personId.js

```diff
@@ -23,17 +23,18 @@ export function uiSchema(
   const fileNumberProp = prefix === 'veteran' ? 'va' : 'relativeVa';
 
   return {
-    [`${prefix}SocialSecurityNumber`]: _.assign(ssnUI, {
-      'ui:required': formData => !_.get(noSSN, formData),
-    }),
+    [`${prefix}SocialSecurityNumber`]: {
+      ...ssnUI,
+      'ui:required': formData => !get(noSSN, formData),
```

Comment:
Unfortunately I don't see a way to eliminate `get` from this file while still allowing for the `noSSN` argument to be a nested selector like that.

Reply:
Yeah, it doesn't look like there's an option for this one

Reply:
One option would be to use the [deconstructPath](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/data/deconstructPath.js) function directly. But then we'd just be replacing one platform data function with another, which would defeat the purpose 🙁